sunkit_image/radial.py

```diff
@@ -577,3 +583,118 @@ def fnrgf(
         data[annulus] = ratio_mix[0] * smap.data[annulus] + ratio_mix[1] * data[annulus]
 
     return sunpy.map.Map(data, smap.meta)
+
+
+@u.quantity_input(application_radius=u.R_sun, vignette=u.R_sun)
+def rhef(
+    smap,
+    radial_bin_edges=None,
+    application_radius=0 * u.R_sun,
+    upsilon=0.35,
+    method="numpy",
+    vignette=1.5 * u.R_sun,
+    do_vignette=None,
```

Comment:
```suggestion
    vignette=None
```
This should be one keyword, "vignette", it defaults to None and if a user passes in a unit-ed value then it gets applied.